src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -247,10 +247,10 @@ namespace JSONHelpers
         return it != end(deviceInfoMap) ? std::optional{ it->second } : std::nullopt;
     }
 
-    std::optional<CustomZoneSetData> FancyZonesData::FindCustomZoneSet(const std::wstring& guuid) const
+    std::optional<CustomZoneSetData> FancyZonesData::FindCustomZoneSet(const std::wstring& guid) const
```

Comment:
I'm pretty sure this is right. There are `uuid`s and there are `guid`s, and they have the same syntax/meaning...